pytissueoptics/scene/viewer/mayavi/MayaviViewer.py

```diff
@@ -35,6 +36,32 @@ def add(self, *solids: 'Solid', representation="wireframe", lineWidth=0.25, show
                 mlab.quiver3d(*mayaviSolid.normals.components, line_width=lineWidth, scale_factor=normalLength,
                               color=(1, 1, 1))
 
+    @staticmethod
+    def addPolygons(polygons: List[Polygon], representation="surface", lineWidth=1, color=(0, 0, 1), showNormals=False):
+        for polygon in polygons:
+            x = [vector.x for vector in polygon.vertices]
+            y = [vector.y for vector in polygon.vertices]
+            z = [vector.z for vector in polygon.vertices]
+            indices = [i for i in zip(*(iter(range(len(x))),) * 3)]
+            mlab.triangular_mesh(x, y, z, indices, representation=representation, line_width=lineWidth, color=color)
+
+    @staticmethod
+    def addPlane(axis="x", value=0,  limits=[[0, 1], [0, 1], [0, 1]], color=(1, 0, 0), opacity=0.25):
+        x, y, z = [], [], []
+        if axis == "x":
+            x = [value, value, value, value]
+            y = [limits[1][0], limits[1][1], limits[1][1], limits[1][0]]
+            z = [limits[2][0], limits[2][0], limits[2][1], limits[2][1]]
+        elif axis == "y":
+            x = [limits[0][0], limits[0][1], limits[0][1], limits[0][0]]
+            y = [value, value, value, value]
+            z = [limits[2][0], limits[2][0], limits[2][1], limits[2][1]]
+        elif axis == "z":
+            x = [limits[0][0], limits[0][1], limits[0][1], limits[0][0]]
+            y = [limits[1][0], limits[1][0], limits[1][1], limits[1][1]]
+            z = [value, value, value, value]
+        mlab.triangular_mesh(x, y, z, ((0, 1, 2), (0, 2, 3)), representation="surface", color=color, opacity=opacity)
+
```

Comment:
Both are not used. I'm also not sure I see the use case. And I'm not sure to understand addPlane's signature (limits looks like a bbox but its a plane...).

Reply:
It was used when I did plane by plane debugging
I allowed to only show partial amount of polygons from a solid and the splitting plane.

Indeed We can remove it I think.

Reply:
https://files.slack.com/files-pri/T03778D6D-F0397SA07KK/image.png

Reply:
To display this: ![image](https://user-images.githubusercontent.com/31829033/160925750-94d0809f-4e91-412e-a558-0667e19d2cdd.png)

Reply:
resolved with 49a93882ae0990de0c66536d4bc9bc48affc017a